virttest/guest_agent.py

```diff
@@ -539,10 +538,10 @@ def set_user_password(self, password, crypted=False, username="root"):
 
         if crypted:
             openssl_cmd = "openssl passwd -crypt %s" % password
-            password = decode_to_text(process.system_output(openssl_cmd)).strip('\n')
+            password = process.system_output(openssl_cmd).strip('\n')
 
         args = {"crypted": crypted, "username": username,
-                "password": base64.b64encode(password)}
+                "password": base64.b64encode(password.encode()).decode()}
```

Comment:
`password = process.system_output(openssl_cmd).strip('\n')` is bytes, you don't need to encode it again.

Reply:
@hereischen I though it should be, but without this encoding, b64encode() would still fail with `a bytes-like object is required, not 'str'`. I don't really understand why, but only this way it can pass.